tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -546,31 +548,21 @@ def config(self, config):
 class TestJSConfigHypothesisClient:
     """Unit tests for the "hypothesisClient" sub-dict of JSConfig."""
 
-    def test_it_contains_one_service_config(self, config):
-        assert len(config["services"]) == 1
-
-    def test_it_includes_the_api_url(self, config):
-        assert config["services"][0]["apiUrl"] == "https://example.com/api/"
-
-    def test_it_includes_the_authority(self, config):
-        assert config["services"][0]["authority"] == "TEST_AUTHORITY"
-
-    def test_it_disables_share_links(self, config):
-        assert not config["services"][0]["enableShareLinks"]
-
-    def test_it_includes_grant_token(
-        self, config, pyramid_request, grant_token_service
-    ):
+    def test_it(self, config, grant_token_service, context, pyramid_request):
         grant_token_service.generate_token.assert_called_with(
             pyramid_request.lti_user.h_user
         )
-        grant_token = config["services"][0]["grantToken"]
-        assert grant_token == grant_token_service.generate_token.return_value
 
-    def test_it_includes_the_group(self, config, context):
-        groups = config["services"][0]["groups"]
-
-        assert groups == [context.h_group.groupid.return_value]
+        assert len(config["services"]) == 1
+        assert config["services"][0] == {
```

Comment:
I think we prefer this style of assertion instead of one per test. 

Of the boolean properties we were only asserting about `enableShareLinks`, added the reset.